executor/ddl.go

```diff
@@ -251,9 +253,26 @@ func isSystemTable(schema, table string) bool {
 	return false
 }
 
+type objectType int
+
+const (
+	tableObject objectType = iota
+	viewObject
+	sequenceObject
+)
+
 func (e *DDLExec) executeDropTableOrView(s *ast.DropTableStmt) error {
+	obt := tableObject
+	if s.IsView {
+		obt = viewObject
+	}
+	return e.dropTableViewSequence(s.Tables, obt, s.IfExists)
+
+}
+
+func (e *DDLExec) dropTableViewSequence(objects []*ast.TableName, obt objectType, ifExists bool) error {
```

Comment:
The naming is weird. How about renaming it to `dropTableObject` and add some comments?

Reply:
nice, addressed.